src/CostFunction3D.cpp

```diff
@@ -806,8 +806,8 @@ void CostFunction3D::calcInnovation()
 
 void CostFunction3D::calcHTranspose(const real* yhat, real* Astate)
 {
-  uint64_t n,m;         // uint64_t
-  uint64_t j,begin,end; // uint32_t
+  uint64_t n,m;       
+  uint64_t j,begin,end;
```

Comment:
It probably does not hurt but is it enough to use `uint32_t` for `j,begin,end` since the `JH` vector is 32-bit integer type?

Reply:
That makes sense. Thanks for your clarification.